portfolio/src/main/java/com/google/sps/servlets/YourBootstrapper.java

```diff
@@ -4,6 +4,7 @@
 import javax.servlet.ServletContextEvent;
 import com.googlecode.objectify.ObjectifyService;
 
+// listener executed by web.xml, required for objectify
```

Comment:
nice! Thanks

Reply:
Maybe name this class something more descriptive then? Like "ObjectifyServletBootstrapper"?

Reply:
done